app/assets/javascripts/discourse/components/second-factor-form.js.es6

```diff
@@ -0,0 +1,36 @@
+import computed from "ember-addons/ember-computed-decorators";
+
```

Comment:
By looking at this code, I don't know that `1` or `2` means.

You should use constants for this:

```
export const SECOND_FACTOR_CODE = 1;
export const SECOND_FACTOR_BACKUP = 2;
```

Then use those constants everywhere that you use `1` and `2` in the files below.